build/ci/jenkins_pipeline_build

```diff
@@ -1,4 +1,11 @@
 pipeline {
+    parameters {
+        string(name: 'IMAGE_VERSION', defaultValue: "1.7.0")
+        string(name: 'OPERATOR_IMAGE', defaultValue: "ibm-block-csi-operator-amd64")
+        string(name: 'DOCKER_REGISTRY', defaultValue: "stg-artifactory.haifa.ibm.com:5030")
```

Comment:
I think we should keep this default value only in our jenkins, since it's internal

Reply:
I think it should rely on an env var, similar to [here](https://stackoverflow.com/a/60287831), since it's an internal value

Reply:
but with an env var, you still writing the internal value in the jenkinsfile, just in another place

Reply:
done